charts/gardener/gardenlet/values.yaml

```diff
@@ -79,7 +79,7 @@ global:
         # - production
         seed:
           concurrentSyncs: 5
-          syncPeriod: 1m
+          syncPeriod: 15m
```

Comment:
I was wondering the same. I will switch it to 1h since we can always restart `gardenlet` locally to test the bootstrap process. And this actually reminds me to add it as a release note in case anyone is depending on those values.